src/OrbitGl/TextRenderer.cpp

```diff
@@ -304,10 +304,19 @@ void TextRenderer::AddText(const char* text, float x, float y, float z, TextForm
   pen_.x = pen_pos[0];
   pen_.y = pen_pos[1];
 
+  const size_t len = strlen(text);
+  if (len == 0) {
+    return;
+  }
```

Comment:
Maybe put this at the top of the function so that we don't call WorldToScreenPos if we don't have to. We should probably use `string_view` in TextRenderer to have the length directly instead of calling strlen, but that's for another PR.

Reply:
nit. `len` is not used, maybe we don't need the variable? `if(strlen(text) == 0) return;`?

Reply:
I'm not sure why I didn't put it at the very top, changed that, thanks for pointing this out. I also removed the `len` variable.